script/Ciel/Day 12/12_01_CIEL12_3.txt

```diff
@@ -261,12 +261,12 @@ Hold on, give me a sec...!"
 [sha:3f96b77c4840f0281e18d98294e4ab0307773cf9]{
 -- Page 90, Offset 31094.
 -- はやる気持ちを抑えられず、あたふたと携帯を操作する。
-Unable to control my excited self, I hastily fumble around on my phone.
+Unable to control my excitement, I hastily fumble around with my phone.
 }
 [sha:eb0bcc884f4a0b942d520ddf7b65e1cb1f6b111e]{
 -- Page 90, Offset 31095.
 -- ……我ながら情けない。生まれつき小市民なせいか、突然の幸福にまったく対応できていない。
-...A pathetic display if I say so myself. Perhaps it's because I was born in the lower middle class, but I can't deal with this sudden influx of happiness whatsoever.
+...A pathetic display, if I do say so myself. Perhaps it's because I'm a born member of the lower middle class, but I have absolutely no idea how to deal with this sudden influx of happiness.
```

Comment:
i cannot even begin to understand this logic

Reply:
![what-did-he-mean-by-this-dove](https://user-images.githubusercontent.com/97076550/183545333-4d5cdf69-423f-4d14-9adb-d087f1c9f310.gif)